phpunit/class-reusable-blocks-render-test.php

```diff
@@ -0,0 +1,97 @@
+<?php
+/**
+ * Reusuable block rendering tests.
+ *
+ * @package Gutenberg
+ */
+
+/**
+ * Tests reusuable block rendering.
+ */
+class Reusuable_Blocks_Render_Test extends WP_UnitTestCase {
```

Comment:
Typo: "Reusuable" -> "Reusable"

Reply:
Thank you. https://github.com/WordPress/gutenberg/pull/4736 corrects my mistake.